identify/extensions.py

```diff
@@ -243,6 +243,7 @@
     'vb': {'text', 'vb'},
     'vbproj': {'text', 'xml', 'vbproj'},
     'vcxproj': {'text', 'xml', 'vcxproj'},
+    'vcxproj.filters': {'text', 'xml', 'vcxproj'},
```

Comment:
this will not work as written

Reply:
Could you elaborate?

Reply:
I guess [this](https://github.com/pre-commit/identify/actions/runs/5222287345/jobs/9427575937?pr=390#step:7:47) is why.